fonts/fonts.css

```diff
@@ -1,8 +1,26 @@
 @font-face {
   font-family: 'Montserrat';
   src:
-    url('montserrat-600-normal.woff2') format('woff2'),
-    url('montserrat-600-normal.woff') format('woff');
-  font-weight: normal;
-  font-style: normal;
+      url('montserrat-semibold.woff2') format('woff2'),
+      url('montserrat-semibold.woff') format('woff');
+      font-weight: 600;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/max_osipov/pullRequest?prid=5654195)